docs/v2/documentation/core-concepts/architecture.mdx

```diff
@@ -149,85 +121,101 @@ Messages are the fundamental units of interaction within sessions. They may also
 - AI responses
 - System notifications
 - Rich media content
+- User actions (clicked, reacted, etc.)
```

Comment:
i feel like having "etc)." and "etc.)" within a few lines of one another is surprisingly jarring, no strong feelings about which to use but it should be consistent